continuous-improvement/designing.md

```diff
@@ -203,7 +203,7 @@ title: Designing content
   </a>
  </li>
  <li>
-  <a href="{{ site.url }}/specificationse/organizing-content.html#user">
+  <a href="{{ site.url }}/specifications/information-findability/organizing-content.html#user)">
```

Comment:
There's a ")" after #user so it isn't directing to the correct spot. Please fix.